### DISCUSSION TITLE
Printing footer only on last page footer

### DISCUSSION THREAD

**Discussion by kittiu** [frappe/print_designer#147] [Ideas]
My case is to print the signature field only on last page. Currently it repeat on every single page. To me, there are 2 feature that can solve this,

1. If I can tell by jinja this is a last page, I can conditionally print the fields.
2. If there is a setting like "Print footer on last page only", this would works too.

For now, are there way to work around this requirement?

![image](https://github.com/frappe/print_designer/assets/1973598/b572babc-ba3d-4cfe-85a7-16e5a2172cb0)

**Comment by kittiu** [comment]
Note: as you may know in other established, there will be 1) page footer and 2) last page footer. This could be the great addition here.

**Reply by maharshivpatel** [reply]
@kittiu for now, whatever is not in the footer area ( box shown in the above image ) will be printed on last page only. 

In the future there will be a possibility to have different header/footer for first/last page.
